provisioners/deploy-backend.yml

```diff
@@ -16,40 +28,36 @@
         job: "cd / && run-parts --report /etc/cron.minute"
     - name: Copy packages from S3
       become_user: deployer
-      command: "aws s3 cp s3://permanent-repos/{{ perm_env }}/{{ item }}.tar.gz /data/tmp/{{ item }}.tar.gz"
-      loop:
-        - api
-        - daemon
-        - library
-        - mdot
-        - task-runner
-        - uploader
-        - upload-service
-        - website
+      command: "aws s3 cp s3://permanent-repos/{{ perm_env }}/{{ item }}.tar.gz {{ temp_dir }}/{{ item }}.tar.gz"
+      loop: "{{ repos }}"
+    - name: Create temp untar dir
```

Comment:
I would have thought this would come before the "copy from S3" command, so the tars have somewhere to land - what am I missing?

Reply:
Looks like the previous command `aws s3 cp` creates that missing directory. We can remove the step that explicitly creates it, since it does nothing according to the ansible log:
```
TASK [Create temp untar dir] ***************************************************
task path: /home/runner/work/infrastructure/infrastructure/provisioners/deploy-cron.yml:28
ok: [ec2-34-220-107-44.***.compute.amazonaws.com] => {"changed": false, "gid": 1005, "group": "deployer", "mode": "0755", "owner": "deployer", "path": "/data/tmp/www", "size": 4096, "state": "directory", "uid": 1005}
```